src/Tribe/REST/V1/Endpoints/Single_Attendee.php

```diff
@@ -92,4 +92,274 @@ public function get( WP_REST_Request $request ) {
 
 		return tribe_attendees( 'restv1' )->by_primary_key( $request['id'] );
 	}
+
+	/**
+	 * Handles POST requests on the endpoint.
+	 *
+	 * @param WP_REST_Request $request
+	 * @param bool            $return_id Whether the created post ID should be returned or the full response object.
+	 *
+	 * @return WP_Error|WP_REST_Response|int An array containing the data on success or a WP_Error instance on failure.
+	 */
+	public function create( WP_REST_Request $request, $return_id = false ) {
+
+		$post_data = $this->prepare_attendee_data( $request );
+
+		if ( is_wp_error( $post_data ) ) {
+			return $post_data;
+		}
+
+		/** @var Tribe__Tickets__Attendees $attendees */
+		$attendees       = tribe( 'tickets.attendees' );
+		$attendee_object = $attendees->create_attendee( $post_data['ticket'], $post_data['data'] );
+
+		if ( ! $attendee_object ) {
+			return new WP_Error( 'attendee-creation-failed', __( 'Something went wrong! Attendee creation failed.', 'event-tickets' ) );
+		}
+
+		$attendee = $post_data['provider']->get_attendee( $attendee_object->ID );
+		$response = new WP_REST_Response( $attendee );
+		$response->set_status( 201 );
+
+		return $response;
+	}
+
+	/**
+	 * Returns the content of the `args` array that should be used to register the endpoint
+	 * with the `register_rest_route` function.
+	 *
+	 * @since TBD
+	 *
+	 * @return array Array of supported arguments for the create endpoint.
+	 */
+	public function CREATE_args() {
```

Comment:
Any reason for the capitalization here? We could make them `get_create_args()` and `get_edit_args()` for clarity.